great-idea-website/css/index.css

```diff
@@ -63,4 +63,127 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+.container {
+	margin: 0 auto;
+	width: 800px;
+	font-size: 20px;
+	height: 150%;
+}
+.top-container {
+display: flex;
+width: 900px;
+margin-bottom: 30px;
+padding-top: 20px;
+}
+nav {
+	font-size: 20px;
+	justify-content: space-between;
+	word-spacing: 60px;
+}
+nav a {
+	justify-content: space-between;
+	width: 900px;
+}
+.logo {
+	display: inline-block;
+	justify-content: end;
+	margin-left: 18px;
+}
+.middle-container {
+	display: flex;
+	width: 130%;
+	justify-content: space-between;
+	border-bottom: 1px solid black;
+	margin-top: 20px;
+	margin-bottom: 20px;
+}
+h1 {
+	margin-top: 15px;
+	font-size: 60px;
+	width: 20px;
+	justify-content: center;
+	margin-left: 60px;
+	padding-left: 25px;
+	padding-top: 30px;
+}
+.img-id1 { 
+justify-items: end;
+justify-items: right;
+margin-bottom: 15px;
+padding-bottom: 20px;
+padding-top: 5px;
+}
+.double-btn {
+	display: inline;
+}
+.btn {
+	height: 25px;
+	font-size: 20px;
+	margin-left: 80px;
+	border: 2px solid black;
+}
+.parent-container {
+	margin: 0 auto;
+	height: 50%;
+	margin-top: 20px;
+}
+.top-content {
+	display: flex;
+	width: 120%;
+	margin-bottom: 15px;
+}
+.inner-1 {
+    line-height: 1.3;
+}
+.inner-2 {
+	justify-content: right;
+	justify-content: end;
+	margin-left: 80px;
+	line-height: 1.3;
+	padding-left: 15px;
+}
+.middle-content {
```

Comment:
If you are listing the margin for more than one side you should do your margins in one line, so for this it would be `margin: 20px 0`